### DISCUSSION TITLE
seq: invalid floating point argument

### DISCUSSION THREAD

**Discussion by MattMHB** [datalust/seq-tickets#2308] [Help]
I'm trying to run seq auth from within a pod in Kubernetes but I am getting this error

[invalid floating point argument](seq: invalid floating point argument: 'auth')

I need to turn off Entra logins so that I can connect using username and password to update an expired client secret for OpenID

**Comment by nblumhardt** [comment]
Hi Matt, thanks for your message.

In the Docker container, the Seq binary is called `seqsvr` to avoid clashing with the `seq` shell built-in.

Let me know if this does the job,
Nick

**Reply by MattMHB** [reply]
Hi, thank you for the reply however I cant stop the service using seqsvr. If I try a seqsvr auth without stopping the service I get an error

**Reply by nblumhardt** [reply]
In Kubernetes, you don't need to stop the pod, but instead need to restart it after creating an init script with the reset command in it:

- https://docs.datalust.co/docs/recovering-from-broken-authentication-settings
- https://docs.datalust.co/docs/init-scripts

Let me know if this helps,
Nick
